src/lib.rs

```diff
@@ -1,17 +1,25 @@
 use rand::{distributions::Alphanumeric, Rng};
 use sha3::{Digest, Keccak256};
 
+// TODO: create fn that accepts salt as &[u8; 32] instead of just &[u8]
+// TODO: rename functions in major release to make sense with previously mentioned fn
+// TODO: should panic/return a Result object if prefix/suffix has incorrect characters
+// TODO: add regex fn for salt generation
+// TODO: convert panics into Results
+// TODO: rename the salt_suffix argument to salt_prefix
```

Comment:
Agree with these adjustments, will try to work on this weekend. I feel some tests doesn't makes sense till we make these adjustments.